lib/1.4/dml-builtins.dml

```diff
@@ -3452,8 +3452,16 @@ nil.
 <small>destroy(void \*data)</small>
 </dt><dd>
 
-This method is called when an event is removed from the event
-queue
+This method is called on any posted events when the device object is deleted.
+<!-- (and also on `SIM_event_cancel_*`, but that function is not exposed for
+custom events so no need to mention) -->
```

Comment:
Heh. Double-check that nothing breaks and that this comment doesn't show up in the rendered documentation.

Reply:
The comment is still there in the generated HTML, but does not get rendered.